### ISSUE TITLE
I have following doubt regarding the code provided

### ISSUE THREAD

**Issue by PalgunaGopireddy** [grip-unina/SAR-CNN#3]
Hi. Thanks for the code. Though you said this is python implementation, not the MATLAB implementation as mentioned in the paper. I have following simple doubts regarding understanding of the code provided.

1.  why log noise is devided by 2 in `PreprocessingLogNoisyFromAmp` class in dataloaders.py and the target image is not?. OfCourse even devided by 2, it is still noise. Just wanted to know is there any particular reason for that.

2. What is the purpose of the `mask`.  I do not see it anywhere mentioned in the paper.

1.  Could you explain the exact purpose of this code in experimental_utility.py, which is used in the train_epoch function? I see that row and column values are used for preparing `target`, `target_amp`, and `mask`. But I am unable to understand the purpose of this code, like why the change values before calculating the loss and metrics.

	        pad_row = (target.shape[2] - pred.shape[2])//2
        	pad_col = (target.shape[3] - pred.shape[3])//2
        	if pad_row > 0:
            		target = target[:, :, pad_row: -pad_row, :]
            		target_amp = target_amp[:, :, pad_row: -pad_row, :]
            		mask = mask[:, :, pad_row: -pad_row, :]
        	if pad_col > 0:
            		target = target[:, :, :, pad_col: -pad_col] #.contiguous()
            		target_amp = target_amp[:, :, :, pad_col: -pad_col] #.contiguous()
            		mask = mask[:, :, :, pad_col: -pad_col].contiguous()

1. 	How does this `loss = softplus(2.0 * diff) / 2.0 + softplus(-2.0 * diff) / 2.0 - 0.693147180559945  # glrt` equal to the loss function given in eq.(1) of the paper.

	Does `glrt` mentioned in comment referring to  `generalized likelihood ratio test`
![image](https://github.com/grip-unina/SAR-CNN/assets/106730853/64b6276e-a390-485c-a96d-79fdc6572e3c)

1. Why one is added to the metrics in the codeline `stats_num[stats_key] = stats_num[stats_key] + 1` in `experimental_utility.py`
